game.js

```diff
@@ -1,6 +1,15 @@
 let playerScore = 0;
 let computerScore = 0;
 
+let resultsContainer = document.createElement("div");
+let bodyElement = document.querySelector("body");
+const playerChoice = document.createElement("h4");
+const computerChoice = document.createElement("h4");
+const compScore = document.createElement("h4");
+const yourScore = document.createElement("h4");
+const gameOvermessage = document.createElement("h4");
```

Comment:
Instead of creating all elements, I suggest to define them on the page beforehand and query for them here.